icon_validator/rules/plugin_validators/regeneration_validator.py

```diff
@@ -110,6 +110,17 @@ def __init__(self, plugin_name: str, plugin_directory: str):
         self.plugin_name = plugin_name
         self.plugin_directory = plugin_directory
 
+    def get_checksum_diff(
+        self,
+        post_regen_checksum_file: Checksum,
+        provided_checksum_file: Checksum,
+    ) -> List[str]:
+        diff = []
+        for key, value in json.loads(provided_checksum_file.to_json()).items():
```

Comment:
do we not need to dive deeper for the `schemas` key? looking at the unit tests we should only be returning the `get_blacklist/schema.py` but we return all of the schemas?

Reply:
👍 I've just pushed a commit reflecting to this adjustment